File: BaekJoon/C++/standard_io/1193/01.py
#%%

X = int(input())

i = 1;

while (X > i):
    X -= i
    i += 1

if (i % 2 == 1):
    print(i + 1 - X, '/', X, sep='');
else:
    print(X, '/', i+1-X, sep='');
# %%

File: BaekJoon/C++/standard_io/1546/01.cpp
#include <algorithm>
#include <iostream>

using namespace std;

int main(void)
{
    int n;
    int max = 0;
    int input[1001] = {};
    double result = 0;
    
    scanf("%d", &n);
    for(int i = 0; i < n; i++)
    {
        cin >> input[i];
        if (input[i] > max) max = input[i];
        result += input[i];
    }
    result = ((result / max) * 100) / n;
    printf("%f", result);

    return 0;
}